### PR TITLE
Update samples

### PR DESCRIPTION
**Pull request by sborisenkox**

### CODE REVIEW

File: src/test/java/SearchServiceFacetsRestrictedValuesTest.java [hunk 1]
```diff
@@ -14,66 +14,66 @@
...
 
-    // [START search_client]
-    private static SearchServiceClient getSearchServiceClient() throws IOException {
-        SearchServiceSettings settings = SearchServiceSettings.newBuilder()
-                .setEndpoint(ENDPOINT)
-                .build();
-        return SearchServiceClient.create(settings);
-    }
-    // [END search_client]
+  // [START search_client]
```
Comment (t-karasova) [inline src/test/java/SearchServiceFacetsRestrictedValuesTest.java:28]: please remove excessive region tags [START/END...] and add them before import and in the end of code sample

File: src/test/java/SearchServiceFilteringTest.java [hunk 2]
```diff
@@ -9,57 +9,57 @@
...
 
-    // [START search_client]
-    private static SearchServiceClient getSearchServiceClient() throws IOException {
-        SearchServiceSettings settings = SearchServiceSettings.newBuilder()
-                .setEndpoint(ENDPOINT)
-                .build();
-        return SearchServiceClient.create(settings);
-    }
-    // [END search_client]
+  // [START search_client]
```
Comment (t-karasova) [inline src/test/java/SearchServiceFilteringTest.java:23]: please remove region tags here and in other places inside the code sample and add region tags before import and in the end of the code sample

File: src/test/java/SearchServiceFilteringTest.java [hunk 3]
```diff
@@ -9,57 +9,57 @@
...
-    @Test
-    public void search() throws IOException, InterruptedException {
+  @Test
+  public void search() throws IOException, InterruptedException {
 
-        searchFilteredProducts("Nest_Maxi", 10,
-                               "(colorFamily: ANY(\"grey\")) AND (price>=100)"
-        );
-    }
+    searchFilteredProducts("Nest_Maxi", 10,
```
Comment (t-karasova) [inline src/test/java/SearchServiceFilteringTest.java:61]: change the values according to the python samples

File: src/test/java/SearchServicePaginationNextPageTest.java [hunk 4]
```diff
@@ -1,3 +1,24 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
```
Comment (t-karasova) [inline src/test/java/SearchServicePaginationNextPageTest.java:2]: please remove these three pagination classes and add one corresponding this https://github.com/t-karasova/grs-samples-python/blob/master/search/search_with_pagination.py one instead

File: src/test/java/SearchUpdateAttributeConfigurationTest.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  // update the product attribute
+  public static void updateProduct(String productToUpdateId) throws IOException {
+
+    // Get a product from catalog
+    Product productToUpdate = getProduct(productToUpdateId);
+
+    // Prepare the product attribute to be updated
+    final CustomAttribute customAttribute = CustomAttribute.newBuilder()
```
Comment (t-karasova) [inline src/test/java/SearchUpdateAttributeConfigurationTest.java:65]: please check the field "attributes" here :https://cloud.google.com/retail/docs/reference/rpc/google.cloud.retail.v2#product

File: src/test/java/SearchServiceFilteringTest.java (hunk 3 above)
Comment (sborisenkox) [reply src/test/java/SearchServiceFilteringTest.java:61]: I don't see an appropriate sample for this file.
It's kind https://github.com/t-karasova/grs-samples-python/blob/master/search/search_with_filtering.py
But not, the file above according https://github.com/t-karasova/grs-retail/blob/update_samples/src/test/java/SearchServiceFacetsExcludedFilterKeyTest.java

For this file, I did refactoring and insert comments.

Comment (t-karasova) [reply src/test/java/SearchServiceFilteringTest.java:80]: this is SearchServiceFilteringTest file and the value of the  filter should be "'(colorFamily: ANY("Black"))'"

I agree that you left the Facet classes with the existing data as there is no such files in python samples yet

File: src/test/java/SearchServiceFilteringTest.java [hunk 6]
```diff
@@ -9,57 +27,60 @@
...
-        searchClient.awaitTermination(2, TimeUnit.SECONDS);
-        System.out.println("Search with filtering, response: " + response);
-        return response;
-    }
-    // [END search_product_filter]
+  // call the Retail Search:
+  @Test
+  public void search() throws IOException, InterruptedException {
+    // TRY DIFFERENT FILTER EXPRESSIONS HERE:
+    String filter = "(colorFamily: ANY(\"grey\")) AND (price>=100)";
```
Comment (t-karasova) [inline src/test/java/SearchServiceFilteringTest.java:78]: capitalize "grey"

File: src/test/java/SearchServiceFilteringTest.java (hunk 3 above)
Comment (sborisenkox) [reply src/test/java/SearchServiceFilteringTest.java:80]: Ok, fixed filter variable.

**Review by t-karasova — APPROVED** [review]

**Pull request merged**